### ISSUE TITLE
Option to change learning rate based on eval data loss?

### ISSUE THREAD

**Issue by ragnarlbrok** [tensorflow/tensor2tensor#1595]
In tensor2tensor is there any option to change learning rate during training based on eval data loss in between training steps? I mean if eval loss in not decreasing or constantly increasing after several steps then can we change learning rate during training based on eval scores . Is there any option for that?
